ms/spaceness.tex

```diff
@@ -106,7 +107,7 @@ \section{Introduction}
 
 To study this, we have implemented an individual-based model in continuous geography 
 that incorporates overlapping generations, local dispersal of offspring, and density-dependent survival. 
-We simulate chromosome-scale genomic data in tens of thousands of individuals from parameter regimes relevant to common subjects of population genetic investigation such as humans and \textit{Drosophila}, and output the full genealogy and recombination history of all final-generation individuals. 
+We simulate chromosome-scale genomic data in tens of thousands of individuals from parameter regimes relevant to common subjects of population genetic investigation, and output the full genealogy and recombination history of all final-generation individuals. 
```

Comment:
(a) there are more species than these two and (b) it's not really clear modern populations of either fall in the range of values we simulated.